src/AWS.Deploy.CLI/ServerMode/Tasks/DeployRecommendationTask.cs

```diff
@@ -30,6 +32,14 @@ public async Task Execute()
             await _orchestrator.DeployRecommendation(_cloudApplication, _selectedRecommendation);
         }
 
+        public async Task<string> CreateCDKDeploymentProject(CdkProjectHandler cdkProjectHandler)
+        {
+            await CreateDeploymentBundle();
```

Comment:
We should just be forwarding into the `Orchestrator` which already has access to the `cdkprojecthandler` and we also don't have to add in `OrchestratorSession`.

Reply:
Done